docs/user-guide/release-notes/2.8.x.rst

```diff
@@ -64,6 +64,12 @@ Bugs
 Known Issues
 ------------
 
+* CentOS 7 users may experience a problem when upgrading. Please refer to note in upgrade
+  instructions for workaround.
+
+* Users that have `/var/lib/pulp` directory symlinked will experience a
```

Comment:
s/that have/that have the/
It also needs to say, "or one of it's subdirectories"
Also there is a sphinx rule that allows for linking by ID only instead of having the entire URL hardcoded in the docs